vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -216,26 +223,26 @@ private VCFHeader getVcfHeaderFilteringInfoLines(String headerObject) throws IOE
         return (VCFHeader) featureCodecHeader.getHeaderValue();
     }
 
-    public VCFHeader getMergedVcfHeader(List<VariantSource> sources, boolean excludeAnnotations) throws IOException {
+    public VCFHeader getMergedVcfHeader(List<VariantSource> sources) throws IOException {
         Map<String, VCFHeader> headers = getVcfHeaders(sources);
 
         Set<VCFHeaderLine> mergedHeaderLines = VCFUtils.smartMergeHeaders(headers.values(), true);
         VCFHeader header = new VCFHeader(mergedHeaderLines, outputSampleNames);
 
-        header = addMissingMetadataLines(header, excludeAnnotations);
+        header = addMissingMetadataLines(header);
 
         return header;
     }
 
-    private VCFHeader addMissingMetadataLines(VCFHeader header, boolean excludeAnnotations) {
+    private VCFHeader addMissingMetadataLines(VCFHeader header) {
         // GT line
-        if (header.getFormatHeaderLine("GT") == null) {
-            header.addMetaDataLine(new VCFFormatHeaderLine("GT", 1, VCFHeaderLineType.String, "Genotype"));
+        if (header.getFormatHeaderLine(GENOTYPE_KEY) == null) {
+            header.addMetaDataLine(new VCFFormatHeaderLine(GENOTYPE_KEY, 1, VCFHeaderLineType.String, "Genotype"));
         }
         if (!excludeAnnotations) {
             // CSQ line
-            if (header.getInfoHeaderLine("CSQ") == null) {
-                header.addMetaDataLine(new VCFInfoHeaderLine("CSQ", 1, VCFHeaderLineType.String,
+            if (header.getInfoHeaderLine(ANNOTATION_KEY) == null) {
```

Comment:
Is CSQ a VEP exclusive tag? If the vcf header stored in the files collection has already a CSQ tag and is different to this one, this may not work as we expected.